Sources/SpeziOnboarding/OnboardingView.swift

```diff
@@ -17,7 +17,9 @@ import SwiftUI
 /// - Tip: The ``SequentialOnboardingView`` provides an alternative to provide
 /// sequential information that is displayed step by step.
 ///
-/// The following example demonstrates the usage of the `OnboardingView` using its default configuration.
+/// The following example demonstrates the usage of the `OnboardingView` using its default configuration. The default configuration divides up
```

Comment:
Please reference the `OnboardingView` via double backticks, that's the DocC norm: https://www.swift.org/documentation/docc/formatting-your-documentation-content